controller/src/action/action_service.rs

```diff
@@ -27,7 +34,24 @@ impl ActionService {
         Self { repository }
     }
 
-    pub async fn create(&self, action_dto: &ActionDTO) {
-        self.repository.create(action_dto.pipeline_id, &action_dto.name, &action_dto.container_uri, &action_dto.r#type, &action_dto.status).await;
+    pub async fn create(
+        &self,
+        action_dto: &ActionDTO,
+        commands: Vec<String>,
+    ) -> Result<Action, sqlx::Error> {
+        self.repository
```

Comment:
You can use `command_service` to create a different command.

Reply:
Ok you made a point